examples/overview/index.rsh

```diff
@@ -1,4 +1,5 @@
 'reach 0.1';
+'use strict';
```

Comment:
This program is referenced in `overview.scrbl`, so the line numbers need to be updated.